.github/workflows/ci.yml

```diff
@@ -63,3 +63,34 @@ jobs:
         run: ./util/prep-bazel-airgapped-build.sh
       - name: Build in the airgapped environment
         run: ./ci/scripts/test-airgapped-build.sh
+
+  verible-lint:
+    name: Verible lint
+    runs-on: ubuntu-20.04
```

Comment:
I'd prefer we don't downgrade OS. The old action runs on ubuntu 24.04 (latest) so please keep it that way.

Reply:
Ah, sure. Would you prefer `-latest` or `-24.04`?

Reply:
Hmm, actually the `-latest` upgrade to 24 is still being rolled out: https://github.com/actions/runner-images/issues/10636.

Let's try ubuntu-24.04 first, if it doesn't work then we can use ubuntu-22.04.

Reply:
Yep, works fine on `ubuntu-24.04`